src/project_manager.ts

```diff
@@ -22,6 +23,11 @@ export default class ProjectManager {
     return this.getProjectInfo().version;
   }
 
+  /** Gets the name of this project. */
+  public static getName(): string {
```

Comment:
what's the point of a public static getter for a public static property?

Reply:
Ok, I made it private.